skip.js

```diff
@@ -0,0 +1,5 @@
+const platform = process.env.npm_config_platform || process.platform
+
+if (platform === 'win32') {
```

Comment:
Shouldn't this if condition be negated, so that it only exits with code 1 when the platform is _not_ win32?